Assets/Scripts/System/SceneJsonReaderSystem.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using Assets.Scripts.EditorState;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using UnityEngine;
+using Zenject;
+
+namespace Assets.Scripts.System
+{
+    public class SceneJsonReaderSystem
+    {
+        private static DecentralandSceneData decentralandSceneData;
+
+        // Dependencies
+        private IPathState pathState;
+
+        [Inject]
+        private void Construct(IPathState pathState)
+        {
+            this.pathState = pathState;
+        }
+
+        /// <summary>
+        /// Loads the Scene Data that the user works on. Loads it just once and stores it as a static variable
+        /// </summary>
+        /// <param name="reload">if it should reload the scene data from the json file again</param>
+        /// <returns>the Decentraland scene data; if there is an error it returns null</returns>
+        [CanBeNull]
+        public DecentralandSceneData GetSceneData(bool reload)
+        {
+            try
+            {
+                if (reload || decentralandSceneData == null)
+                {
+                    var json = File.ReadAllText(pathState.ProjectPath + "/scene.json");
+                    decentralandSceneData = JsonConvert.DeserializeObject<DecentralandSceneData>(json);
+                }
+            }
+            catch (Exception e)
+            {
+                Debug.LogError("Error while reading scene.json");
+                Debug.LogException(e);
+                return null;
+            }
+
+            return decentralandSceneData;
+        }
+
+        /// <summary>
+        /// Stores the Decentraland Scene Data
+        /// </summary>
+        public class DecentralandSceneData
+        {
+            public Display display { get; set; }
+            public string owner { get; set; }
+            public Contact contact { get; set; }
+
+            public string main { get; set; }
+
+            public List<string> tags { get; set; }
+
+            public Scene scene { get; set; }
+
+            public List<SpawnPoints> spawnPoints { get; set; }
+
+            public List<string> requiredPermissions { get; set; }
+
+            public FeatureToggles featureToggles { get; set; }
+
+            public class Display
+            {
+                public string title { get; set; }
+                public string description { get; set; }
+                public string navmapThumbnail { get; set; }
+                public string favicon { get; set; }
+            }
+
+            public class Contact
+            {
+                public string name { get; set; }
+                public string email { get; set; }
+            }
+
+            public class Scene
+            {
+                public List<string> parcels { get; set; }
+
+                public string @base { get; set; }
```

Comment:
`base` is a reserved word. Because of that, it can't be used as a name for a field. But the json file requires this name for the field. The solution is to add an `@`. That means writing `@base` acts like a field with the name `base`. 
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim